dags/helpers/email_triggers.py

```
@@ -0,0 +1,478 @@
+    # Format duration if available
+    if duration_raw and isinstance(duration_raw, (int, float)):
+        duration = _format_duration(float(duration_raw))
+    else:
+        duration = 'N/A'
+    
+    # Get the full traceback if available
+    try:
+        full_traceback = traceback.format_exc()
+        if full_traceback == 'NoneType: None\n':  # Common case when exception is passed directly
+            full_traceback = str(exception)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)